x-team/auto-ui
# Add new line when is updating a note

**Release Type:** *Bug fix* 


Fixes https://x-team-internal.atlassian.net/browse/XP-2647 

## Description

> Added correct value with break line when is the note is updated.



## Checklist

**Before submitting a pull request,** please make sure the following is done:



## Related PRs
*N/A*


## Steps to Test or Reproduce
1- Go to the admin page
2- in the note section, create and update notes
3- Line break should be persistent in the create and update note.



## Impacted Areas in Application

- Note section



## Screenshots
![update_note_new_line](https://user-images.githubusercontent.com/5392202/49826660-d1e48380-fd5d-11e8-8825-021ea0193af8.gif)
